ludwig/features/image_feature.py

```diff
@@ -313,13 +335,16 @@ def add_feature_data(
 
                 proc_df[feature[PROC_COLUMN]] = backend.df_engine.map_objects(
                     input_df[feature[COLUMN]],
-                    lambda file_path: read_image_and_resize(get_abs_path(src_path, file_path))
+                    lambda img_store: read_image_and_resize(
+                        get_abs_path(src_path, img_store))
+                    if isinstance(img_store, str) else img_store
```

Comment:
shouldn't this be `else read_image_and_resize(img_store)`?
Also this is a pretty long lambda, maybe we can define a litle local helper function for clarity ;)

Reply:
Good catch...you're correct I missed this one.  
created helper function `_get_processed_image()`